sirepo/package_data/static/json/raydata-schema.json

```diff
@@ -58,14 +58,19 @@
         "planMetadata": {
             "notes": ["Notes", "Text", ""]
 	},
+	"pollBlueskyForScansAnimation": {
+	    "minutes": ["Minutes to poll for", "Integer", 1, "Number of minutes to poll for new scans", 1, 1440]
```

Comment:
Perhaps we want use use units like we do elsewhere eg. `Poll Interval [min]`

Reply:
2eacb899b I don't like interval because to me that means "poll for new scans once every n mins" which isn't what is going on.